src/test/java/seedu/address/logic/parser/EditCommandParserTest.java

```diff
@@ -180,19 +166,19 @@ public void parse_multipleRepeatedFields_failure() {
         assertParseFailure(parser, userInput, Messages.getErrorMessageForDuplicatePrefixes(PREFIX_PHONE));
 
         // mulltiple valid fields repeated
-        userInput = targetIndex.getOneBased() + PHONE_DESC_AMY + ADDRESS_DESC_AMY + EMAIL_DESC_AMY
-                + TAG_DESC_FRIEND + PHONE_DESC_AMY + ADDRESS_DESC_AMY + EMAIL_DESC_AMY + TAG_DESC_FRIEND
-                + PHONE_DESC_BOB + ADDRESS_DESC_BOB + EMAIL_DESC_BOB + TAG_DESC_HUSBAND;
```

Comment:
I believe this is the part where we made use of Bob's email description. Was the removal on purpose to keep the test shorter but still valid? 🤔